src/Pages/Login/Login.scss

```
@@ -0,0 +1,181 @@
+.Login {
+  width: 1000px;
+  margin: 0 auto;
+  margin-top: 80px;
+  padding: 25px;
+  font-size: 14px;
+
+  h1 {
+    text-align: center;
+    font-size: 25px;
+  }
+
+  .loginContainer {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 50px;
+
+    .loginCont {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
```

```suggestion
    .loginCont {
      margin: 0 auto;
```
같은 목적을 달성할 수 있을것 같네요!

---

스타일을 변경을 해봤는데 스타일 적용이 잘안됩니다..